website/screens/components/date-input/code/examples/optional.ts

```diff
@@ -1,14 +1,10 @@
 import { DxcDateInput, DxcInset } from "@dxc-technology/halstack-react";
 
 const code = `() => {
-    return(
-        <DxcInset space="large">
-            <DxcDateInput
-            label="Default"
-            placeholder="Placeholder"
-            optional
-            />
-        </DxcInset>
+  return (
+    <DxcInset space="large">
+      <DxcDateInput label="Default" placeholder="Placeholder" optional />
```

Comment:
I don't think this example gives any kind of value or extra explanation to the user... Maybe we should remove it 🤔.

Reply:
I think it helps some teams since many of them are more used to required components than optional, but I also see that it could be unnecessary if they read all the props.

But I like to give a little bit more attention to this.